backend/src/channel-message/dto/update-channel-message.input.ts

```diff
@@ -0,0 +1,11 @@
+import { Field, InputType } from '@nestjs/graphql'
+import { IsNotEmpty, IsString, MaxLength } from 'class-validator'
+
+@InputType()
+export class UpdateChannelMessageInput {
```

Comment:
We should use 'extends PartialType' no ?

Reply:
yes